.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 orbs:
-  ike: bartoszmajsak/ike@0.0.1
+  ike: aslakknutsen/ike@0.0.2
```

Comment:
You can switch back to upstream if you want :) Just pushed new version with your fixes.